apps/central-scan/frontend/src/components/delete_batch_modal.tsx

```diff
@@ -25,10 +25,10 @@ export function DeleteBatchModal({
   return (
     <Modal
       onOverlayClick={onClose}
-      title={`Delete ‘${batchLabel}’?`}
+      title={`Delete ‘${batchLabel}’`}
```

Comment:
I remember we use "Remove" a lot in VxAdmin, should we use that here?

Reply:
I'm fine using delete here. Although we're not perfectly consistent, in many case "Remove" on VxAdmin refers to removing files that originated someone and thus can be reloaded. In the scanners, the data originates on the machine and deleting clears the original.

Reply:
sgtm